Cargo.toml

```diff
@@ -9,12 +9,16 @@ license = "Apache-2.0/MIT"
 keywords = ["math", "random"]
 authors = ["The Noise-rs Developers."]
 
+[profile.release]
```

Comment:
Do we want to leave this in?  Enabling debug in release builds is useful for debugging and benchmarking, but it can create release binaries that are very large.

Also, I don't actually know if this has any effect for libraries.

Reply:
Definitely didn't mean to commit that, feel free to remove.